src/Symfony/Bridge/Twig/Mime/BodyRenderer.php

```diff
@@ -22,9 +22,9 @@
  */
 final class BodyRenderer implements BodyRendererInterface
 {
-    private $twig;
-    private $context;
-    private $converter;
+    private Environment $twig;
+    private array $context;
+    private HtmlConverter $converter;
```

Comment:
I'd mark that nullable, as it is an optional dependency and so not having a converter is a normal state

Reply:
Leaving it uninitialized also works well in this case, but I see your point. Now that the PR is merged, would you like to change the property to nullable? I'd accept a PR with that change. 🙂